game/combo.go

```diff
@@ -48,6 +48,7 @@ type Board interface {
 	Get(Position) (Square, error)
 	AvailableMoves(Color) []Move
 	IfMove(Move) Board
+	RemainingPieces(Color) int
```

Comment:
Not sure I would expand the Board interface with this. The Board interface is mostly for (built-in) players to use for convenience, and this probably wouldn't be used by players (or they would just count it themselves since they need to look at all the squares anyway). I think it makes more sense to implement on top of Board via a utility method (same could be said for AvailableMoves(), but I moved it onto board so I could make it faster by skipping method calls).